tools/validators/instance_validator/generate_universe.py

```diff
@@ -0,0 +1,63 @@
+import sys
```

Comment:
same comments as on the previous PR regarding the import

Reply:
Done